internal/cli/apps.go

```diff
@@ -119,11 +119,11 @@ var (
 		ShortForm: "r",
 		Help:      "Display the application secrets ('signing_keys', 'client_secret') as part of the command output.",
 	}
-	number = Flag{
+	appNumber = Flag{
 		Name:      "Number",
 		LongForm:  "number",
 		ShortForm: "n",
-		Help:      "Number of apps to retrieve",
+		Help:      "Number of apps, that match the search criteria, to retrieve. Maximum result number is 1000.",
```

Comment:
```suggestion
		Help:      "Number of apps to retrieve. (Default 50, min 1, max 1000)",
```

Reply:
I'm seeing that all the descriptions you've provided match the same pattern. I would avoid using the phrase "that match the search criteria" because it suggests some queryable properties to search against. While that mechanism exists for a select number of resources, it is not true of all. Let's not overcomplicate the phrasing here.

Reply:
That's fair, I'll move them all to match the help text you've suggested above

Reply:
As cobra/pflags generates a `(default is 50)` for each of these I went with what is in d8c07f88597419f779b354ed158439265d41613a to avoid having a double bracket and double default value.

To demonstrate the difference (not sure whether to remove the final `.` in the text
`-n, --number int   Number of organizations to retrieve. (Default 50, min 1, max 1000) (default 50)`
vs
`-n, --number int   Number of permissions to retrieve. Minimum 1, maximum 1000. (default 50)`